expression/builtin_time_vec.go

```diff
@@ -470,11 +470,88 @@ func (b *builtinNullTimeDiffSig) vecEvalDuration(input *chunk.Chunk, result *chu
 }
 
 func (b *builtinGetFormatSig) vectorized() bool {
-	return false
+	return true
 }
 
+// vecEvalString evals a builtinGetFormatSig.
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_get-format
 func (b *builtinGetFormatSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf0, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf0)
+	if err = b.args[0].VecEvalString(b.ctx, input, buf0); err != nil {
+		return err
+	}
+	buf1, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf1)
+	if err = b.args[1].VecEvalString(b.ctx, input, buf1); err != nil {
+		return err
+	}
+
+	result.ReserveString(n)
+	for i := 0; i < n; i++ {
+		if buf0.IsNull(i) || buf1.IsNull(i) {
+			result.AppendNull()
+			continue
+		}
+		format := buf0.GetString(i)
+		location := buf1.GetString(i)
+		res := b.getFormat(format, location)
+		result.AppendString(res)
+	}
+	return nil
+}
+
+func (b *builtinGetFormatSig) getFormat(format, location string) string {
```

Comment:
Please make the row-oriented implementation also use this function.

Reply:
Rest, LGTM

Reply:
And please resolve conflicts, PTAL ~ @b41sh

Reply:
ok